sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -4670,7 +4670,7 @@ public void onTimer(
     }
 
     @Test
-    @Category(NeedsRunner.class)
+    @Category({NeedsRunner.class, UsesTimersInParDo.class, UsesTestStream.class})
```

Comment:
Thanks a lot for adding all of these categories. It will be helpful. Maybe add `UsesTestStream.class` to all of those that also have `UsesTestStreamWithProcessingTime.class`?

Reply:
Good point, this is what I would like to understand the common practice in beam.

`UsesTestStreamWithProcessingTime` is a subtype of `UsesTestStream`, based on JUnit documentation, tests marked as `UsesTestStreamWithProcessingTime` is also part of `UsesTestStream` but not the other way around.

Therefore, if a test is marked as `UsesTestStreamWithProcessingTime`, then marking it with `UsesTestStream` does not effectively change how it behaves since it transiently belongs to `UsesTestStream` category as well.

However, I do agree that having both annotations here are clearer and more readable and there does not seem to have cons to have both annotations.

What are your suggestions here?

Reply:
Updated all tests with `UsesTestStreamWithProcessingTime` to have `UsesTestStream` for now.

Reply:
hi - well, I am not an expert on JUnit, but we have recently started storing the annotations, and we plan over the long term to use them to build a capability matrix, so I think it's best to separate all the sub-categories. Thanks!

Reply:
I have updated the PR so that tests with `UsesTestStreamWithProcessingTime` are also annotated with `UsesTestStream`, does this look good to you?